src/ui/testrtc-test.html

```diff
@@ -0,0 +1,180 @@
+<!--
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<dom-module id="testrtc-test">
+  <style>
+    .test-output {
+      margin: 0 2em 0 2em;
+      word-wrap: break-word;
+    }
+
+    paper-progress {
+      display: block;
+      min-width: 2em;
+    }
+
+    .header {
+      background-color: transparent;
+      height: 2em;
+      min-width: 80%;
+      display: flex;
+      padding-left: 16px;
```

Comment:
Please explain? What is possible to use here? Styling? Yes by styling on the dom-module the style only applies inside here and is isolated from other elements.